input/fsh/Terminology.fsh

```diff
@@ -266,7 +266,6 @@ Title: "Surgery of Axilla"
 Description: "Valueset of surgery types of the axilla"
 * SCT#396487001 "Sentinel lymph node biopsy"
 * SCT#234262008 "Excision of axillary lymph node"
-* SCT#79544006 "Complete axillary lymphadenectomy"
```

Comment:
Why is this one removed?

Reply:
I removed the clearance option from this VS, because we already have the clearance profile. And when it's due to lymph node involvement this can be added in the reasonReference.